### PR TITLE
feat: apply api call to CollectorScheduleForm

### PR DESCRIPTION
**Pull request by WANZARGEN**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci` ONLY)
- [ ] Not that difficult

### Type of Change
- [ ] New feature
- [ ] Bug fixes
- [x] Feature improvement
- [x] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Affects to
- [ ] Packages
  - [ ] core-lib
  - [ ] mirinae
  - [ ] etc 

- [ ] Apps
  - [ ] storybook
  - [x] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?

### Description

Please review in order by commits!

### CODE REVIEW

**Review by yuda110 — APPROVED** [review]
LGTM!! I really read it.

File: apps/web/src/services/asset-inventory/collector/collector-detail/CollectorDetailPage.vue [hunk 1]
```diff
@@ -124,8 +123,6 @@ const state = reactive({
     loading: true,
     collector: computed<CollectorModel|null>(() => collectorFormState.originCollector),
     collectorName: computed<string>(() => state.collector?.name ?? ''),
-    // TODO: must be updated after backend api spec is updated
-    collectorProviders: computed<undefined|string[]>(() => (state.collector?.provider ? [state.collector.provider] : undefined)),
```
Comment (skdud4659) [inline apps/web/src/services/asset-inventory/collector/collector-detail/CollectorDetailPage.vue:128]: Well done~!

**Review by sulmoJ — APPROVED** [review]
LGTM~

**Review by skdud4659 — APPROVED** [review]
LGTM~!🥳

**Pull request merged**